tests/test_transformations.py

```diff
@@ -0,0 +1,177 @@
+from pathlib import Path
+import math
+import random
+import os
+
+import pytest
+import h5py
+import numpy as np
+import cv2
+import imageio
+
+import segmentation_labeling_app.transforms.transformations as transformations
+
+
+@pytest.fixture()
+def video_fixture():
```

Comment:
We don't need to use a real video file. This one is quite big for a test resource, and we don't know exactly what's in it so it's hard to make the `expected` result. Just generate a small sample 3d numpy array such that it's easy to test (e.g. binary values, `n` * 2 * 2 dimensions.

Reply:
Sounds good, moved to a in memory movie. I'll remove the file from the commit histroy.